freeipa/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClientBuilder.java

```diff
@@ -226,4 +232,16 @@ private CloseableHttpResponse execute(HttpPost post, CookieStore cookieStore) th
     private static HostnameVerifier hostnameVerifier() {
         return (s, sslSession) -> true;
     }
+
+    private static JsonRpcClient.RequestListener noOpRequestListener() {
```

Comment:
we could avoid this noop implementation if we don't set anything if it's not necessary. Also the JsonRpcClient can handle null as RequestListener

Reply:
ah good point didn't realize null can be handled